src/components/About/Blurbs.js

```diff
@@ -71,7 +79,7 @@ class Blurbs extends Component {
             And that's the story of how fortissimo was conceived.
           </h3>
 
-          <img className = "RightBlurbImages" src={require("../../img/img-logo-colored.jpg")} alt="fortissimo-logo"/ >
+          <img className = "fillRightBlurbImages" src={require("../../img/img-logo-colored.jpg")} alt="fortissimo-logo"/ >
 
         </div>
       </div>
```

Comment:
About/Blurbs.js: Code is clean. Good use of LeftCircleDiv, RightCircleDiv to hold images and fillLeftBlurbImages, fillRightBlurbImages to present images.